config/stack/manifests/app.yaml

```diff
@@ -1,5 +1,5 @@
 # Human readable title of application.
-title: Crossplane Azure
+title: Crossplane Azure Provider
 
 overviewShort: The Azure Crossplane Stack enables resource management with the Microsft Azure Cloud Platform.
```

Comment:
want to make other changes to this file to go from `Azure Crossplane Stack` -> `Azure Crossplane Provider`?